packages/orbit-components/src/tmp-tailwind/Radio/index.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import cx from "clsx";
+import React from "react";
+
+import cloneWithTooltip from "../../utils/cloneWithTooltip";
+import getFieldDataState from "../../common/getFieldDataState";
+import type { Props } from "./types";
+
+const Radio = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
+  const {
+    label,
+    value,
+    hasError = false,
+    disabled = false,
+    checked = false,
+    onChange,
+    name,
+    info,
+    readOnly,
+    id,
+    tabIndex = 0,
+    dataTest,
+    tooltip,
+  } = props;
+  const [isFocused, setFocused] = React.useState(false);
+
+  return (
+    <label htmlFor={id} className="font-base flex w-full cursor-pointer items-start">
+      <input
```

Comment:
The styles applied to descendants on hover and active can be applied as follows (example only):

```
    &:active ${StyledIconContainer} {
      border-color: ${theme.orbit.paletteBlueNormal};
      transform: ${`scale(${theme.orbit.modifierScaleCheckboxRadioActive})`};
    }
```

can be transformed into

```
  "[&_.orbit-radio-icon-container]:active:border-blue-normal [&_.orbit-radio-icon-container]:active:scale-95"
```

Notice that I am NOT considering disabled, so please do it. It was just an example on how to target a different element while considering pseudo-states on this element.

Reply:
It works ok on applied on the descendant, even better